Puppetfile

```diff
@@ -7,25 +7,23 @@ moduledir File.join(File.dirname(__FILE__), 'modules')
 # Core modules used by 'apply'
 mod 'puppetlabs-service', '0.6.0'
 mod 'puppetlabs-facts', '0.5.1'
-mod 'puppet_agent',
```

Comment:
I have not had a chance to verify the new `puppet_agent::install` task parameters https://github.com/puppetlabs/puppetlabs-puppet_agent/commit/358add8dfaff091f4f9346e47652fbdb764cb909#diff-11a53f781e47039ffcac93b25abd1e81

Reply:
I think shipping these is low risk. Acceptance tests for 2.1.1 are green and while there are not yet tests for the new parameters they are unlikely to cause regressions.